### PR TITLE
Employees

### PR DESCRIPTION
**Pull request by edodoe15**
Endpoints for retrieving, adding, deleting and updating employees

### CODE REVIEW

File: Employees/add_employee_serializer.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Ant-Adis) [inline Employees/add_employee_serializer.py]: There are too many empty lines, can you remove it here and in the rest of he PR

Comment (edodoe15) [reply Employees/add_employee_serializer.py]: Thanks for advice, I have edited code.

**Review by Ant-Adis — APPROVED** [review]

**Comment by Ant-Adis** [comment]
You have some conflicts in PR, please resolve it

**Comment by edodoe15** [comment]
Resolved 👍

**Pull request merged**